db/migrate/20171101152316_create_admins.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAdmins < ActiveRecord::Migration
+  def change
+    create_table :admins do |t|
+      t.string :name, null: false
```

Comment:
doesn't change anything but `t.string :name, null: false, index: {unique: true}` and we can eliminate line 8